receiver/sqlqueryreceiver/config.go

```diff
@@ -81,6 +82,8 @@ type MetricCfg struct {
 	Unit             string            `mapstructure:"unit"`
 	Description      string            `mapstructure:"description"`
 	StaticAttributes map[string]string `mapstructure:"static_attributes"`
+	StartTsColumn    string            `mapstructure:"start_ts_column"`
+	StopTsColumn     string            `mapstructure:"stop_ts_column"`
```

Comment:
Are you sure that `stop_ts_column` is the right name for this setting? Isn't it more like a "current" timestamp?

Reply:
Depending upon the received rows stop may not necessarily be the current timestamp. I agree `stop_ts_column` is not the right name.  I am changing it to `end_ts_column`.